app/screens/send-bitcoin-screen/suggestion-modal.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react"
+import { makeStyles, useTheme } from "@rneui/themed"
+import { View, TextInput } from "react-native"
+import { StackNavigationProp } from "@react-navigation/stack"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { testProps } from "../../utils/testProps"
+import { RootStackParamList } from "@app/navigation/stack-param-lists"
+import CustomModal from "../../components/custom-modal/custom-modal"
+import { useFeedbackSubmitMutation } from "@app/graphql/generated"
+import { gql } from "@apollo/client"
+
+gql`
+  mutation feedbackSubmit($input: FeedbackSubmitInput!) {
+    feedbackSubmit(input: $input) {
+      errors {
+        message
+        __typename
+      }
+      success
+      __typename
+    }
+  }
+`
+
+export const SuggestionModal: React.FC<{
+  navigation: StackNavigationProp<RootStackParamList>
+  showSuggestionModal: boolean
+  setShowSuggestionModal: (value: boolean) => void
+}> = ({ navigation, showSuggestionModal, setShowSuggestionModal }) => {
+  const { LL } = useI18nContext()
+  const styles = useStyles()
+  const {
+    theme: { colors },
+  } = useTheme()
+  const [suggestion, setSuggestion] = React.useState("")
+  const [sendFeedback] = useFeedbackSubmitMutation()
+
+  const submitSuggestion = async () => {
+    await sendFeedback({ variables: { input: { feedback: suggestion } } })
+    setShowSuggestionModal(false)
+    navigation.popToTop()
+  }
+
+  const dismissSuggestionModal = () => {
+    navigation.popToTop()
+    setShowSuggestionModal(false)
+  }
+
+  return (
+    <CustomModal
+      title={LL.support.thankYouText()}
+      minHeight={"50%"}
+      titleMaxWidth={"100%"}
+      titleTextAlignment={"left"}
+      toggleModal={dismissSuggestionModal}
+      isVisible={showSuggestionModal}
+      primaryButtonTitle={LL.common.submit()}
+      primaryButtonOnPress={submitSuggestion}
+      body={
+        <View style={styles.field}>
+          <TextInput
+            {...testProps(LL.SendBitcoinScreen.suggestionInput())}
+            style={styles.noteInput}
+            onChangeText={(improvement: React.SetStateAction<string>) =>
+              setSuggestion(improvement)
+            }
+            placeholder={LL.SendBitcoinScreen.suggestionInput()}
+            placeholderTextColor={colors.grey2}
+            value={suggestion}
+            multiline={true}
+            numberOfLines={3}
+            autoFocus
+          />
+        </View>
+      }
+      secondaryButtonTitle={LL.AuthenticationScreen.skip()}
+      secondaryButtonOnPress={dismissSuggestionModal}
+      showCloseIconButton={false}
+    />
+  )
+}
+
+const useStyles = makeStyles(({ colors }) => ({
+  noteInput: {
+    color: colors.black,
+  },
+  field: {
+    padding: 10,
+    marginTop: 10,
```

Comment:
`marginTop` and `marginBottom` may not be necessary here because it is already provided by the CustomModal.